lib/jobs/upsert-node-job.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+module.exports = UpsertNodeAttributeFactory;
+
+var di = require('di');
+
+di.annotate(UpsertNodeAttributeFactory, new di.Provide('Job.Upsert.Node.Attribute'));
+di.annotate(UpsertNodeAttributeFactory, new di.Inject(
+    'Job.Base',
+    'Util',
+    'Services.Waterline',
+    'Logger'
+    )
+);
+
+function UpsertNodeAttributeFactory(BaseJob, util, waterline, Logger) {
+    var logger = Logger.initialize(UpsertNodeAttributeFactory);
+
+    function UpsertNodeAttribute(options, context, taskId) {
+        UpsertNodeAttribute.super_.call(this, logger, options, context, taskId);
+        this.nodeId = this.options.target || context.target;
+        this.data = this.options.data;
+    }
+    util.inherits(UpsertNodeAttribute, BaseJob);
+    
+    UpsertNodeAttribute.prototype._run = function run() {
+        var self = this;
+        return waterline.nodes.updateByIdentifier(self.nodeId, self.data) 
```

Comment:
That seems like a big ask of the user to take on the responsibility of ensuring our database model stays consistent.  

We are going through a huge effort to add schema validation around tasks.  Providing an unvalidated entry point to modifying the nodes collection seems counter to that ongoing effort.  

We just removed this type of raw patching methodology in favor of the OBM API in 2.0, so I'm not sure why we are regressing back to this methodology.  The use case linked in PT should be resolved with a standard API entry point (ie: OBM APIs) and the poller creation API.  Assuming you don't want the user to use the OBM API (and/or the REST API task to do it), maybe you want something like this https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/create-ipmi-obm-settings.js that is specific (and can be validated with a schema) to what you referenced in PT and includes methods to validate the user supplied settings function before committing them to the database.

@yyscamper @RackHD/corecommitters thoughts on this?